components/LibraryIndexDisplay/LibraryIndexDisplay.jsx

```diff
@@ -25,7 +26,11 @@ function LibraryIndexDisplay(props) {
 			}
 		}
 		fetchLibraryItems()
-	}, [])
+	}, [handleError])
+
+	const handleNewClick = () => {
+		navigate('/library/new')
```

Comment:
Is this meant to route to /library/new, we don't currently have this route defined in App.jsx

Reply:
Hey Will, yes this intentional. Because the route isn't defined, it will route to /library/:itemId which will then check what the ID is. If the ID is 'new' it will perform differently which is how we control between a new form and an edit form in the same component.

Reply:
Thank you for clarifying - approved!